main.tex

```diff
@@ -342,7 +341,7 @@ \subsection{Relation to the VO}
 
 Some general VO tools are available in astroquery.  The \texttt{vo\_conesearch}
 package, which originally resided in \astropy, is now part of \astroquery.  VO
-Cone Search has \texttt{query\_*\_region} interface like the other \astroquery
+Cone Search has a \texttt{query\_region} interface like the other \astroquery
```

Comment:
I think that `query_region` is meant because `query_region_async` does not match the `*` pattern.